libcontainer/nsenter/cloned_binary.c

```diff
@@ -65,19 +65,20 @@
 #  else
 /* These values come from <https://fedora.juszkiewicz.com.pl/syscalls.html>. */
 #    warning "libc is outdated -- using hard-coded SYS_memfd_create"
-#    if defined(__x86_64__) // x86_64
+#    if defined(__x86_64__)	// x86_64
 #      define SYS_memfd_create 319
-#    elif defined(__i386__) // i386
+#    elif defined(__i386__)	// i386
 #      define SYS_memfd_create 356
-#    elif defined(__ia64__) // ia64
+#    elif defined(__ia64__)	// ia64
 #      define SYS_memfd_create 1340
-#    elif defined(__arm__) // arm
+#    elif defined(__arm__)	// arm
 #      define SYS_memfd_create 385
-#    elif defined(__aarch64__) // arm64
+#    elif defined(__aarch64__)	// arm64
```

Comment:
I should probbably point this out as well, as it's difficult to see on GitHub;

```c
#    elif defined(__aarch64__)	// arm64
  ^                           ^
  spaces                      tab
```

Perhaps that's standard, but it felt a bit odd that the code uses tabs (both for indentation _and_ for aligning?), but in these blocks, _spaces_ are used for indentation, which felt a bit odd (perhaps it's convention/standard?). Of course, I'm also used to Go's formatting now (tabs for indentation, but spaces for aligning).

Reply:
Spaces are used to indent the nested preprocessor directives as described above; let me quote myself

> Adds a few more options to indent. This was mostly motivated by
trying to save the existing formatting as much as possible, minimizing
the amount of changes indent produces.
> ...
> * -ppi2: sets the indentation for nested preprocessor directives
to 2 spaces (same as in current code defining SYS_memfd_create);

Current code use ~three~ four different ways to indent the nested cpp directives
(no indentation, tabs, two spaces, spaces and tabs). I have settled on two spaces.

As I said the idea was to minimize the amount of changes, and yet keep the formatting
consistent. I will be happy to reconsider (there's also an option to ignore cpp indentation).